br/pkg/restore/client.go

```diff
@@ -2535,6 +2536,35 @@ func (rc *Client) UpdateSchemaVersion(ctx context.Context) error {
 	return nil
 }
 
+const (
+	alterTableDropIndexFormat  = "ALTER TABLE `%s`.`%s` DROP INDEX `%s`;"
+	alterTableAddIndexFormat   = "ALTER TABLE `%s`.`%s` ADD INDEX `%s`(%s);"
+	alterTableAddPrimaryFormat = "ALTER TABLE `%s`.`%s` ADD PRIMARY KEY (%s)"
```

Comment:
here, currently, do not support clusterd PK online changed. also should care able add unique index.